addons/web/static/src/legacy/js/core/menu.js

```diff
@@ -235,7 +260,15 @@ export async function initAutoMoreMenu(el, options) {
         return Promise.all(defs);
     }
 
+    function _getExtraMenuEl() {
+        return el.querySelector(".o_extra_menu_items .dropdown-toggle");
+    }
+
     function _endAutoMoreMenu() {
+        const extraMenuEl = _getExtraMenuEl();
+        if (extraMenuEl && isExtraMenuOpen) {
+            extraMenuEl.click();
```

Comment:
Can't you call the bootstrap function to open it instead of clicking?

Reply:
Yes, it's the right thing to do, but just to be cautious with the use of this code in the non-lazy context, I used the `click()` instead :+1: